flake.nix

```diff
@@ -12,10 +12,16 @@
     };
     crane = {
       url = "github:ipetkov/crane";
-      inputs.nixpkgs.follows = "nixpkgs";
     };
     topiary = {
-      url = "github:tweag/topiary";
+      # We pin this specific commit because it contains the latest change to
+      # Nickel formatting, and doesn't contain the feature of "dynamically
```

Comment:
Since the "let blocks" change, I think cff73fc no longer contains the latest changes. I guess we can deal with that later.

Reply:
Yes, especially because of dynamic grammar loading, I anticipate that we'll node more than just a revision bump (not necessarily much more, but use a different output of the Topiary flake, etc.)